torch/monitor/__init__.py

```diff
@@ -1 +1,32 @@
 from torch._C._monitor import *  # noqa: F403
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from torch.utils.tensorboard import SummaryWriter
+
+
+STAT_EVENT = "torch.monitor.Stat"
+
+class TensorboardEventHandler:
+    """
+    TensorboardEventHandler is an event handler that will write known events to
+    the provided SummaryWriter.
```

Comment:
we should also mention it only handles scalars

Reply:
did we update this? :)

Reply:
see the next line :)